packages/ui/src/components/va-viewer/VaViewer.stories.ts

```diff
@@ -1,12 +1,96 @@
 import { defineComponent } from 'vue'
-import VaViewer from './VaViewer.demo.vue'
+import { VaViewer } from './'
+import { VaImage } from '../va-image'
+import { VaIcon } from '../va-icon'
+import { userEvent } from '@storybook/testing-library'
+import { expect } from '@storybook/jest'
+import { sleep } from '../../utils/sleep'
 
 export default {
   title: 'VaViewer',
   component: VaViewer,
+  tags: ['autodocs'],
 }
 
-export const Default = defineComponent({
-  components: { VaViewer },
-  template: '<VaViewer/>',
+const getImagePath = (width, height = 0) => `https://picsum.photos/id/450/${width}/${height || width}`
+
+export const Default = () => ({
+  components: { VaViewer, VaImage },
+  data: () => ({ getImagePath }),
+  template: `
+    <va-viewer>
+      <va-image
+        :src="getImagePath(600, 800)"
+        :max-width="200"
+      >
+        <template #loader>
+          Loading...
+        </template>
+      </va-image>
+    </va-viewer>
+  `,
+})
+
+Default.play = async ({ canvasElement, step }) => {
+  const viewer = canvasElement.querySelector('.va-viewer')
+  await step('Click on image to show a magnified view', async () => {
+    const beforeClick = canvasElement.querySelectorAll('.va-viewer-content')
+    expect(beforeClick.length).toBe(0)
+    userEvent.click(viewer)
+    await sleep()
+    const afterClick = canvasElement.querySelectorAll('.va-viewer-content')
+    expect(afterClick.length).toBe(1)
```

Comment:
I don't know why this won't work. I have tested with document.querySelector function and it doesn't work either.

Reply:
Actucally document.querySelector worked just fine for me.

The issue was that viewer spawned outside of canvasElement.

<img width="1030" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/7cd12fb9-b2d3-4d51-910a-2595c9c744af">